draft-ietf-scitt-architecture.md

```diff
@@ -68,6 +68,14 @@ contributor:
     contribution: >
       Orie contributed to improving the generalization of COSE building blocks and document consistency.
 
+  - ins: D. Brooks
+    name: Dick Brooks
+    organization: Business Cyber Guardian (TM)
+    email: dick@businesscyberguardian.com
+    country: United States
+    contribution: >
+      Dick contributed some software supply chain use cases consistent with US DHS CISA Secure by Design principles and practices
```

Comment:
Reducing the text a bit to focus on the IETF doc content.
```suggestion
      Dick contributed to the software supply chain use cases.
```

Reply:
No objection Steve.